### ISSUE TITLE
Issue with "extends": (short summary here please)

### ISSUE THREAD

**Issue by alattalatta** [mdn/content#9195]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/extends

#### What information was incorrect, unhelpful, or incomplete?

I can't understand what _the extension_ means in this context:

> The `.prototype` of the extension must be an {{jsxref("Object")}} or {{jsxref("null")}}.

My guess is that this...

```js
function Super() {}
Super.prototype = 3

class Child extends Super {}
```

...isn't valid? (`Uncaught TypeError: Class extends value does not have valid prototype property 3`)

#### Specific section or headline?

[Description](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/extends#description)

#### What did you expect to see?

Clearer description; there's no mention of the word *extension* in the spec.

#### Did you test this? If so, how?

No.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/javascript/reference/classes/extends`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/extends
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/classes/extends/index.md
* Last commit: https://github.com/mdn/content/commit/cf607d68522cd35ee7670782d3ee3a361eaef2e4
* Document last modified: 2021-07-20T18:47:57.000Z

</details>

**Comment by wbamberg** [comment]
I agree "extension" is not clear at all. If I understand you right, do you think "parent" would be a good replacement for "extension" here?

**Comment by alattalatta** [comment]
That would be better. But for me, "The `.prototype`"  was also quite hard to comprehend what exactly it refers to, especially since I don't modify the `prototype` property nowadays. It would help greatly with just a few code examples.
